machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -34,6 +34,10 @@ include::modules/machineset-yaml-azure.adoc[leveloffset=+3]
 
 Machine sets running on Azure support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-non-guaranteed-instance_creating-machineset-azure[Spot VMs]. You can save on costs by using Spot VMs at a lower price compared to standard VMs on Azure. You can xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-creating-non-guaranteed-instance_creating-machineset-azure[configure Spot VMs] by adding `spotVMOptions` to the `MachineSet` YAML file.
 
+include::modules/machineset-yaml-azure-stack-hub.adoc[leveloffset=+3]
+
+Machine sets running on Azure Stack Hub support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-non-guaranteed-instance_creating-machineset-azure[Spot VMs]. You can save on costs by using Spot VMs at a lower price compared to standard VMs on Azure Stack Hub. You can xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-creating-non-guaranteed-instance_creating-machineset-azure[configure Spot VMs] by adding `spotVMOptions` to the `MachineSet` YAML file.
```

Comment:
Spot instances are not supported on ASH at all, they are missing from this platform, is it possible to leave a note about it?